docs/docs/FAQ.mdx

```diff
@@ -230,14 +232,19 @@ to increase the bar for what the algorithm considers a "core face" for that pers
 
 ### The immich_model-cache volume takes up a lot of space, what could be the problem?
 
-If you installed several models and chose not to use some of them, it might be worth deleting the old models that are in immich_model-cache.
+If you installed several models and chose not to use some of them, it might be worth deleting the old models that are in immich_model-cache. To do this you can mount the model cache and remove the undesired models.
 
-To do this you can run:
+<details>
+<summary>Steps</summary>
+
+```bash
+docker run -it --rm -v immich_model-cache:/mnt-models alpine sh
+cd /mnt-models
+ls
+# delete unused models with 'rm -r <model_name>'
```

Comment:
The models are under `clip` and `facial-recognition`, so they should either `cd` to those folders first or adjust the `rm -r` command to include that part of the path.